HKImport/Importer.swift

```diff
@@ -43,6 +43,21 @@ class HealthRecord: CustomStringConvertible {
     var metadata: [String: Any]?
 }
 
+extension DateFormatter {
+    convenience init(dateFormat: String) {
+        self.init()
+        self.dateFormat = dateFormat
+    }
+}
+
+extension NumberFormatter {
+    convenience init(locale: Locale, numberStyle: NumberFormatter.Style) {
+        self.init()
+        self.locale = locale
+        self.numberStyle = numberStyle
+    }
+}
```

Comment:
I created these to simplify the `Importer` `init` function and keep from having to wait for `init` completion in tests. On lines 71 and 72 we can directly create Number and Date formatters instead of creating them in `Importer` `init`.